lib/elixir_analyzer/exercise_test/assert_call.ex

```diff
@@ -34,6 +34,8 @@ defmodule ElixirAnalyzer.ExerciseTest.AssertCall do
   end
 
   defp parse(description, block, should_call) do
+    :ok = validate_call_block(block, should_call)
```

Comment:
can we move this call to before lines `29` and `33`? Then we can pass the macro_name as an argument rather than using `should_call` and then coupling the macro_names in a deeper function?

Reply:
Good idea: [`95b170a` (#66)](https://github.com/exercism/elixir-analyzer/pull/66/commits/95b170a2b5398fed085b1052875b4cffcc53094b)